specs/yoga/clusterapi-driver.rst

```diff
@@ -0,0 +1,200 @@
+..
+   This work is licensed under a Creative Commons Attribution 3.0 Unported
+ License.
+
+ http://creativecommons.org/licenses/by/3.0/legalcode
+
+..
+   This template should be in ReSTructured text. The filename in the git
+ repository should match the launchpad URL, for example a URL of
+ https://blueprints.launchpad.net/magnum/+spec/awesome-thing should be named
+ awesome-thing.rst .  Please do not delete any of the sections in this
+ template.  If you have nothing to say for a whole section, just write: None
+ For help with syntax, see http://sphinx-doc.org/rest.html
+ To test out your formatting, see http://www.tele3.cz/jbar/rest/rest.html
+
+==================
+Cluster API driver
```

Comment:
We should really do these review comments in gerrit.

Reply:
Yes, agreed, but I wanted to iterate here first to knock things into order first!  Thanks @JohnGarbutt for the review comments - I'll iterate and squash for Gerrit next.